source/graphics/libbmp.cpp

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2016 - 2017 Marc Volker Dickmann
+ * Project: LibBMP
+ */
+// Modified by Nekiro
```

Comment:
you forgot to remove credits from this file like you just did with 1:1 copied sprite_appearances :D

Reply:
You sure?
https://github.com/mehah/otclient/blob/main/src/client/spriteappearances.cpp#L2

Reply:
> You sure? https://github.com/mehah/otclient/blob/main/src/client/spriteappearances.cpp#L2
> 
> Yep, it is useless, just pointing out hyprocrisy. Cheers.

Point out something useful, if you want to comment just to talk zucchini, better not come. Or did you forget that there in the otland repository it was spoken said that "there is no credit in open source?" Hypocrisy doesn't come from me. Don't charge for what you don't do and don't dirty the pull with silly talk. From the moment the code is implemented in the project, its license becomes valid. If you don't want them to "copy" something of yours, don't open source it, or make a lib integrated into cmake for projects to add as something apart. But, come to think of it, you don't like cmake right? And it is not possible to use this with vcproj.

Reply:
10+ years of open source contribution and Nekiro still doesn't know how licenses and copyrights work. Adding your name to a file that is inside a project with open source license doesn't make it yours, even if you wrote it. Everything added to the project falls down to the original license and is part of the original owner copyright. Anyone can use it without needing to keep your copyrights (but the original one, yes). We remove it to keep it consistent, but we do keep the credits in the PR because we think it's important to have the recognition and proper credits. We can chat privatly if you need any extra info.